agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -2383,21 +2383,25 @@ bool slave_thread::handle_cmdu_ap_manager_message(
         client_association_event_tlv->association_event() =
             wfa_map::tlvClientAssociationEvent::CLIENT_HAS_LEFT_THE_BSS;
 
-        // Add vendor specific tlv
-        auto vs_tlv =
-            message_com::add_vs_tlv<beerocks_message::tlvVsClientAssociationEvent>(cmdu_tx);
+        if (config.non_intel) {
+            LOG(DEBUG) << "non-Intel, skipping ClientAssociationEvent TLV";
```

Comment:
It seems like you're not skipping the whole ClientAssociationEven TLV here, but only the VS part which is added afterwards.
If it's the case, the current debug message is confusing and should rather say something like "non-Intel, not adding the VS clientAssociationEvent TLV", or at least mention that it's the VS part that is being skipped.